src/lib.rs

```diff
@@ -60,16 +86,44 @@ pub fn build_enclaves(args: BuildEnclavesArgs) -> NitroCliResult<()> {
     Ok(())
 }
 
+/// Initializes a temporary directory containing the blobs needed for converting a docker to
+/// an EIF file.
+fn init_blobs(resources_dir: &PathBuf) -> NitroCliResult<TempDir> {
```

Comment:
Why do we need to do this? Can't we read the blobs directly from `resources_dir`?